client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -1,36 +1,163 @@
 /** @format */
 
+/** @TODO remove */
+/* eslint-disable no-console */
+
 /**
  * External dependencies
  */
 import { Component, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
 import { withInstanceId } from '@wordpress/compose';
 
+/**
+ * Internal dependencies
+ */
+import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from 'lib/simple-payments/constants';
+
+/**
+ * @FIXME: toFixed should be replaced with proper decimal calculations. See simple-payments/form
+ * const { precision } = getCurrencyDefaults( values.currency );
+ */
+
 class Edit extends Component {
-	handleChange = event => {
-		const paymentId = parseInt( event.target.value, 10 );
-		if ( paymentId ) {
-			this.props.setAttributes( { paymentId } );
+	componentDidUpdate( prevProps ) {
+		// Save on deselect
+		// @TODO: Add save on post save?
+		// @FIXME: Do not allow multiple saves to be in flight
+		if ( prevProps.isSelected && ! this.props.isSelected ) {
+			this.savePayment();
+		}
+	}
+
+	savePayment = async () => {
+		const { attributes, setAttributes } = this.props;
+		const { paymentId } = attributes;
+
+		// @TODO field validation
+		const path = `/wp/v2/${ SIMPLE_PAYMENTS_PRODUCT_POST_TYPE }/${ paymentId ? paymentId : '' }`;
+		console.log( path );
+
+		try {
+			const { id } = await apiFetch( {
+				path,
+				method: 'POST',
+				body: JSON.stringify( attributesToPost( attributes ) ),
```

Comment:
This is sending the wrong content type. We need to send `application/json`, so this line has to be:
```js
data: attributesToPost( attributes ),
```